flutter_vlc_player/example/lib/vlc_player_with_controls.dart

```diff
@@ -110,13 +118,15 @@ class VlcPlayerWithControlsState extends State<VlcPlayerWithControls>
           }
         }
       }
+      // ignore: no-empty-block
       setState(() {});
```

Comment:
There isn't a since `await` in this async method -- let's make it synchronous.
Let's also move the logic above inside the `setState` callback, per [doc recommendations](https://api.flutter.dev/flutter/widgets/State/setState.html).